homeassistant/components/flume/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "domain": "flume",
+    "name": "Flume",
+    "documentation": "https://www.home-assistant.io/integrations/flume/",
+    "requirements": [
+        "pytz==2019.2",
```

Comment:
No need to add pytz to requirements. It's already a core requirement.

Reply:
Done.